lib/extensions/ar_virtual.rb

```diff
@@ -400,12 +400,15 @@ def virtual_attribute_names
       end
     end
 
-    def attributes_builder
-      @attributes_builder ||= ::ActiveRecord::AttributeSet::Builder.new(attribute_types, primary_key) do |name|
-        unless columns_hash.key?(name) || virtual_attribute?(name)
```

Comment:
`unless ||`  🏃 🙀

Reply:
@jrafanie this was by me, and yes, it was intentional.  This is patching over the original Rails method that only had `unless columns_hash.key(name)` at the time, and for easier comparison with our `|| virtual_attribute?(name)` addition, I left it as the less pretty `unless ||` you speak of.

In less words:  #dealWithIt™

Reply:
Also... https://github.com/ManageIQ/manageiq/pull/13302#discussion_r95384117

Reply:
haha, yeah, now I remember.  I only say it because I literally have to make mental truth tables with `unless` `||` so they might save real estate but I can't grok them easily.